.devcontainer/Dockerfile

```diff
@@ -4,7 +4,6 @@
 #-------------------------------------------------------------------------------------------------------------
 
 ARG NODE_VERSION=12
-ARG USERNAME=node
```

Comment:
Not really an arg...  (the user name is already hardcoded to 'node' in the base image.)